index.html

```diff
@@ -1,417 +1,245 @@
+        <p class="description-text">Hello I’m a software developer! I can help you build a product , feature or website Look
+          through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+          to contact me.</p>
+        <button type="button" name="button" class="enable-button basic-button">Get My Resume</button>
+        <hr class="h-rule">
+        <ul class="skills ul-list vertical-list">
+          <li class="list-item-text languages-skills">
+            <span class="sublist-title">Languages</span>
+            <ul class="ul-list ul-sublist vertical-list">
+              <li class="list-item-text">Javascript</li>
+              <li class="list-item-text">Ruby</li>
+              <li class="list-item-text">Html</li>
+              <li class="list-item-text">Css</li>
+            </ul>
+          </li>
+          <li class="list-item-text frameworks-skills">
+            <span class="sublist-title">Frameworks</span>
+            <ul class="ul-list ul-sublist vertical-list">
+              <li class="list-item-text">Bootstrap</li>
+              <li class="list-item-text">Ruby on Rails</li>
+              <li class="list-item-text">RSPec</li>
+              <li class="list-item-text">CodyBara</li>
+              <li class="list-item-text">Selenium</li>
+            </ul>
+          </li>
+          <li class="list-item-text additional-skills">
+            <span class="sublist-title">Skills</span>
+            <ul class="ul-list ul-sublist vertical-list">
+              <li class="list-item-text">Codekit</li>
+              <li class="list-item-text">Github</li>
+              <li class="list-item-text">Codepen</li>
+              <li class="list-item-text">Gitlab</li>
+              <li class="list-item-text">Terminal</li>
             </ul>
-            <hr class="bg-2-footer">
-        </footer>
-
-
-
-
+          </li>
+        </ul>
+      </section>
+    <section class="page-section contact-section">
+        <section class="contact-form" id="contact-section">
+            <form action="https://formspree.io/f/mrgjeazo" class="contact-form" id="form" method="post">
+              <div class="description-text">
+                <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+              </div> 
+              <ul class="form-widgets ul-list vertical-list">
+                <li><input type="text" name="full-name" maxlength="30" placeholder="Full name" required></li>
+                <li><input type="email" name="email-address" placeholder="Email address" required></li>
+                <li><textarea name="message" maxlength="500" placeholder="Enter text here" required></textarea></li>
+                <li><button type="submit" name="sumbit-button" class="enable-button submit-button">Get in touch</button></li>
+              </ul>
+            </form>
+        </section> 
```

Comment:
The width of your contact section is smaller than the Figma design, which is causing the letters to look very close to one another. Please adjust the width of this section to follow the design:
Also, note that your `h2` color is blue when it should be black (#262626 is the color code on Figma😉).

Your design:
![image](https://user-images.githubusercontent.com/42799579/193387768-281b922c-9ce3-4d48-a41a-1c6a2bca7ea4.png)

Figma design:
![image](https://user-images.githubusercontent.com/42799579/193387602-730420d0-1e30-4d50-8c44-4ce1e848efc6.png)

Reply:
- [ OPTIONAL] The same here you still need to improve your `font-weight` and use the one provided by figma design , kindly fix this for your project to look professional :upside_down_face: